README.md

```diff
@@ -41,6 +41,20 @@ The currently supported component extension types are:
 * `tor-client`
 * `local-data-files` (formerly `tracking-protection`)
 
+### NTP SI component
+
+To pacakge NTP SI components, download assets from from passed url at first. It will download assets to `./build/ntp-sponsored-images/resources/`
+
+```bash
+npm run generate-ntp-sponsored-images -- --data-url <s3 buckets url>
+```
+
+Then, packaging assets to crx files per region. It will generate component crx files for each region at `./build/ntp-sponsored-images/output`. Passed args to `--keys-directory` should include all PEM files that has private key for supported regions. 
```

Comment:
I think "packaging" should be "package"?

Reply:
Done.